### ISSUE TITLE
update Azure based on built-in advisor recommendations

### ISSUE THREAD

**Issue by awentzel** [Microsoft/fast-dna#866]
Azure is in need of some service updates:

- 4 high priority security updates
- 3 medium priority data protection issues for storage of blob data
- 2 low priority upgraded service recommendations

![screen shot 2018-09-10 at 10 39 18 am](https://user-images.githubusercontent.com/16669785/45314130-d17de780-b4e5-11e8-862d-bc650909f554.png)

**Comment by awentzel** [comment]
High priority/medium priority issues have been resolved.  Waiting on services to resolve for the current changes before executing on the low priority issues.

**Comment by awentzel** [comment]
Issue filed on Azure: 118092119065070 : advising on more than 3 owners.

**Comment by awentzel** [comment]
Configured IP addresses across 20 web applications

**Comment by awentzel** [comment]
Delete Ai resource group. We will rebuild this from scratch next round.

**Comment by awentzel** [comment]
Issue moved to [Microsoft/fast-tracking #168](https://github.com/Microsoft/fast-tracking/issues/168) via [**ZenHub**](https://www.zenhub.com/)

**Comment by awentzel** [comment]
Issue moved to [Microsoft/fast-tracking #222](https://github.com/Microsoft/fast-tracking/issues/222) via [**ZenHub**](https://www.zenhub.com/)

**Comment by awentzel** [comment]
Issue moved to [Microsoft/fast-tracking #223](https://github.com/Microsoft/fast-tracking/issues/223) via [**ZenHub**](https://www.zenhub.com/)
